### DISCUSSION TITLE
Roborock integration and map customization

### DISCUSSION THREAD

**Discussion by lageorgem** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#559] [Q&A]
Hello,

I have just gotten a Roborock Qrevo S, which seems to only work with the Roborock app, not the Mi Home app.
I have successfully got it to work with the [xiaomi-vacuum-map-card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card), and it integrates well with [humbertogontijo/homeassistant-roborock](https://github.com/humbertogontijo/homeassistant-roborock).
My main issues are around the map itself: I already got my map looking the way I want it to using the Xiaomi Cloud Map Extractor on my previous Roborock model, and it seems that this new one isn't customizable. At the very least I'd like to be able to rotate it, but it seems aside from cropping there is no other customization on this card's side, while the Roborock integration does not allow map customization at all.

Did anyone have this problem? Any workarounds?

Thanks

**Comment by PiotrMachowski** [comment]
I think this is a question that should be asked in [humbertogontijo/homeassistant-roborock](https://github.com/humbertogontijo/homeassistant-roborock) repo ;)
